Stack/Opc.Ua.Core/Security/Certificates/SecurityConfiguration.cs

```diff
@@ -59,9 +59,40 @@ public void Validate()
             {
                 throw ServiceResultException.Create(StatusCodes.BadConfigurationError, "ApplicationCertificate must be specified.");
             }
+            // ensure mandatory stores are valid
+            ValidateStore(TrustedIssuerCertificates, nameof(TrustedIssuerCertificates));
+            ValidateStore(TrustedPeerCertificates, nameof(TrustedPeerCertificates));
+
+            //ensure optional stores are valid if specified
+            if (TrustedHttpsCertificates != null)
+            {
+                ValidateStore(TrustedHttpsCertificates, nameof(TrustedHttpsCertificates));
+            }
+            if (HttpsIssuerCertificates != null)
+            {
+                ValidateStore(HttpsIssuerCertificates, nameof(HttpsIssuerCertificates));
+            }
+            if (TrustedUserCertificates != null)
+            {
+                ValidateStore(TrustedUserCertificates, nameof(TrustedUserCertificates));
+            }
+            if (UserIssuerCertificates != null)
+            {
+                ValidateStore(UserIssuerCertificates, nameof(UserIssuerCertificates));
+            }
 
-            TrustedIssuerCertificates = CreateDefaultTrustList(TrustedIssuerCertificates);
-            TrustedPeerCertificates = CreateDefaultTrustList(TrustedPeerCertificates);
```

Comment:
By removing calls to CreateDefaultTrustList the method itself is unused and candidate to be removed.